core/soc/estimator.go

```diff
@@ -104,6 +104,19 @@ func (s *Estimator) RemainingChargeEnergy(targetSoC int) float64 {
 	return whRemaining / 1e3
 }
 
+// Range replaces the api.Vehicle.Range interface to take the estimated soc into account
```

Comment:
Ergänzung:

    Range must be called after `SoC` as it relies on the updated soc.